arbnode/node.go

```diff
@@ -1009,9 +1014,26 @@ func createNodeImpl(
 		dasLifecycleManager,
 		classicOutbox,
 		configFetcher,
+		ctx,
 	}, nil
 }
 
+func (n *Node) OnConfigReload(old *Config, new *Config) error {
+	if old.DelayedSequencer.Enable != new.DelayedSequencer.Enable {
+		if new.DelayedSequencer.Enable {
+			delayedSequencer, err := NewDelayedSequencer(n.L1Reader, n.InboxReader, n.TxStreamer, n.SeqCoordinator, func() *DelayedSequencerConfig { return &(n.configFetcher.Get().DelayedSequencer) })
```

Comment:
A simpler mechanism might be to just always create the delayed sequencer if the sequencer is enabled. That way, we don't have to dynamically create it when it gets enabled, and it won't do anything if it isn't enabled.